src/components/async-resource/index.js

```diff
@@ -17,7 +17,12 @@ const AsyncResource = ({
   loading,
   resource,
   resourceParams,
+  disableSync,
```

Comment:
Can't see that disableSync is used anywhere

Reply:
It does not seem so anymore. Removed.